src/schema.rs

```diff
@@ -521,6 +528,34 @@ pub(crate) struct TeamPeople {
     pub include_all_alumni: bool,
 }
 
+#[derive(serde::Deserialize, Clone, Debug)]
+#[serde(remote = "Self")]
+pub(crate) struct TeamMember {
+    #[serde(rename = "name")]
+    pub github: String,
```

Comment:
I know I suggested calling this "name", but I do wonder if using "github" as the key would be better. This is what it's called in the people toml config whereas "name" is used for the person's preferred name.